src/main/java/seedu/address/logic/commands/employee/EditCommand.java

```diff
@@ -119,15 +126,20 @@ private static Employee createEditedEmployee(Employee employeeToEdit,
         Set<Tag> updatedTags = editEmployeeDescriptor.getTags()
                 .orElse(employeeToEdit.getTags());
         EmployeeId updatedEmployeeId = editEmployeeDescriptor.getEmployeeId().orElse(employeeToEdit.getEmployeeId());
-        EmployeePosition updatedEmployeePosition = editEmployeeDescriptor.getEmployeePosition()
-                .orElse(employeeToEdit.getEmployeePosition());
+        EmployeePay updatedEmployeePay = editEmployeeDescriptor.getEmployeePay()
+                .orElse(employeeToEdit.getEmployeePay());
+        EmployeePendingPay updatedEmployeePendingPay = editEmployeeDescriptor.getEmployeePendingPay()
```

Comment:
Revert as EmployeePendingPay and EmployeeTotalSalary should be abstracted into the EmployeeFinances Container